benchmark/src/main/java/com/datatorrent/benchmark/window/README.md

```diff
@@ -0,0 +1,29 @@
+Windowed Operator Benchmark Readme
+
+Introduction
+The Windowed operator benchmark are applications to test the performance of windowed operator. 
+This benchmark applications only available for Malhar 3.7.0 or later versions. 
+The source code is located inside the malhar/benchmark. There are two applications, 
+one is benchmark for WindowedOperator, another is for KeyedWindowedOperator. 
+
+Steps of compile
+- Get source code of malhar
+- Compile the malhar with all module option "-Pall-modules": "mvn clean install -Pall-modules -DskipTests"
+
+Run the benchmark and check the result in cluster
+- Start apex cli
+- Launch application "WindowedOperatorBenchmarkApp" or "KeyedWindowedOperatorBenchmarkApp". Get application id, for example "application_1487803614053_10401"
+- Use command "connect" to connect to the application
+- Use command "list-operators" to list operators of the application
+- Find the operator with name "output". Get the value of "host" field. for example "node1:8041"
+- Go to the resource manager and click "Nodes" link in the left panel
+- Find the node by "Node Address" which run the operator "output", click the link of "Node HTTP Address" column. for example "node1:8042"
+- Expand the "Tools" on the left panel and click "Local logs"
+- Click the "container/" link
+- Select the application folder. The folder name is same as the application id
+- Click the container and then click the log file 
+- Open the log. you will find the benchmark which log in the log file for example: 
+  "total: count: 7356000; time: 278453; average: 26417; period: count: 400000; time: 10005; average: 39980"
```

Comment:
Not asking to remove any logging. Just wondering if you want to mention the number of periods also in the summary? And if the benchmark itself is equipped to find the stable result, then mention it as well.